### ISSUE TITLE
update pillow to fix voulnerability issue

### ISSUE THREAD

**Issue by HomoPolyethylen** [qbic-pipelines/querynator#26]
### Description of the bug

biocontainers hat detected a [vulnerability in querynator 0.4.2](https://quay.io/repository/biocontainers/querynator/manifest/sha256:2394d92709fe5cc02002d39c1065f3b01f056e383a4a9668a07a81c89fed1cb9?tab=vulnerabilities&fixable=true)
should be fixable by updating `pillow` from `10.0.1` -> `10.2.0`

**Comment by HomoPolyethylen** [comment]
command used to check dependencies using mamba:
```bash
mamba repoquery whoneeds pillow
```

**Comment by HomoPolyethylen** [comment]
bumping pillow to 10.2.0 worked using mamba
- [ ] try with conda on thanos

**Comment by HomoPolyethylen** [comment]
```
mamba create -n qn2 querynator -y && mamba activate qn2 && mamba install pillow=10.2.0 --dry-run
```
output:
```
Looking for: ['pillow=10.2.0']

conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
bioconda/linux-64                                           Using cache
bioconda/noarch                                             Using cache
pkgs/r/linux-64                                               No change
pkgs/main/noarch                                              No change
pkgs/main/linux-64                                            No change
pkgs/r/noarch                                                 No change

Pinned packages:
  - python 3.10.*


Transaction

  Prefix: /home/casimir/opt/miniforge3/envs/qn2

  Updating specs:

   - pillow=10.2.0
   - ca-certificates
   - certifi
   - openssl


  Package          Version  Build            Channel           Size
─────────────────────────────────────────────────────────────────────
  Install:
─────────────────────────────────────────────────────────────────────

  + zlib            1.2.13  hd590300_5       conda-forge     Cached
  + jpeg                9e  h166bdaf_2       conda-forge     Cached

  Upgrade:
─────────────────────────────────────────────────────────────────────

  - pillow          10.0.1  py310h29da1c1_1  conda-forge     Cached
  + pillow          10.2.0  py310h5eee18b_0  pkgs/main       Cached

  Downgrade:
─────────────────────────────────────────────────────────────────────

  - libjpeg-turbo  2.1.5.1  hd590300_1       conda-forge     Cached
  + libjpeg-turbo    2.1.4  h166bdaf_0       conda-forge     Cached
  - libtiff          4.6.0  h8b53f26_0       conda-forge     Cached
  + libtiff          4.2.0  hf544144_3       conda-forge     Cached
  - lcms2             2.15  h7f713cb_2       conda-forge     Cached
  + lcms2             2.12  hddcbb42_0       conda-forge     Cached
  - openjpeg         2.5.2  h488ebb8_0       conda-forge     Cached
  + openjpeg         2.4.0  hb52868f_1       conda-forge     Cached

  Summary:

  Install: 2 packages
  Upgrade: 1 packages
  Downgrade: 4 packages

  Total download: 0 B

─────────────────────────────────────────────────────────────────────


Dry run. Exiting.

DryRunExit: Dry run. Exiting.

```

matplotlib untouched, executing as described above works, test queries &report creation on querynator work

**Comment by HomoPolyethylen** [comment]
conda on the other hand has some issues:

```
(qn2) [zxmgc83@thanos querynator]$ conda install pillow=10.2.0
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
done

## Package Plan ##

  environment location: /home-link/zxmgc83/miniconda3/envs/qn2

  added / updated specs:
    - pillow=10.2.0


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    querynator-0.1.3           |     pyhdfd78af_0          18 KB  bioconda
    ------------------------------------------------------------
                                           Total:          18 KB

The following packages will be REMOVED:

  civicpy-3.0.0-py310hdfd78af_0
  pysam-0.22.0-py310h41dec4a_1
  vcfpy-0.13.8-pyhdfd78af_0

The following packages will be UPDATED:

  lcms2                                     2.15-h7f713cb_2 --> 2.16-hb7c19ff_0 
  libdeflate                                1.18-h0b41bf4_0 --> 1.20-hd590300_0 
  libjpeg-turbo                          2.1.5.1-hd590300_1 --> 3.0.0-hd590300_1 
  libtiff                                  4.6.0-h8b53f26_0 --> 4.6.0-h1dd3fc0_3 
  pillow                             10.0.1-py310h29da1c1_1 --> 10.2.0-py310h01dd4db_0 

The following packages will be DOWNGRADED:

  querynator                             0.4.2-pyh7cba7a3_0 --> 0.1.3-pyhdfd78af_0
  ```

**Comment by HomoPolyethylen** [comment]
make environment.yml with mamba, create new env from that using conda, check if this works
```
(qn2) [zxmgc83@thanos querynator]$ conda env create --file qn-mamba.yml 
Channels:
 - conda-forge
 - bioconda
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed
Channels:
 - conda-forge
 - bioconda
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package pillow-10.2.0-py310h5eee18b_0 is excluded by strict repo priority
  ```

=> `channel_priority: strict` in `~/.condarc` makes the difference
-> remove line and conda behaves as mamba

**Comment by HomoPolyethylen** [comment]
issue resolved with release of querynator 0.5.0
https://quay.io/repository/biocontainers/querynator/manifest/sha256:d61912ea181d9395da28725e7ff34245b8a1597b61bc77ef6e0dc88544521926?tab=vulnerabilities
